### DISCUSSION TITLE
optional target for oc modules install-base (et al?)

### DISCUSSION THREAD

**Discussion by iceback** [KarchinLab/open-cravat#78] [Q&A] [answered]
We're in an odd corner where we don't have root access and yet need to install packages.  python itself offers a "--target" option for install.   Is there anything similar for oc module ?

We don't want to force each of us to do the installation.

**Comment by rkimoakbioinformatics** [comment, marked as answer]
You can set the new modules dir with

>oc config md /path/to/common/modules_directory

Then, your `oc` commands will use the new modules directory (under which all modules will be installed).

With the upcoming release of OpenCRAVAT 2.2.6, `--md` option can be given to `oc run`, `oc module install`, etc., so that

>oc module install --md /path/to/common/modules_directory
>oc run --md /path/to/common/modules_directory
Etc.

**Comment by iceback** [comment]
Thank you
